submissions/webdevtoday/task_js_dom/js/Controller.js

```diff
@@ -0,0 +1,27 @@
+import {Render} from './Render.js';
+
+export class Controller {
+    constructor(root) {
+        this.root = root;
+        this.render = new Render( this.root );
+    }
+
+    get(controllerName) {
+        if (!controllerName) {
+            this.render.render( `<h1>404 Page Not Found</h1>` );
+            return false;
+        }
+        controllerName = controllerName[0].toUpperCase() + controllerName.slice(1).toLowerCase();
+        import(`./controllers/${controllerName}Controller.js`)
+            .then(controller => {
+                console.log('then');
+                const content = new controller.default();
+                this.render.render(content.get());
+            })
+            .catch(err => {
+                console.log('catch');
```

Comment:
you can combine these `console.log` and also you can use `console.error`

Reply:
I changed it.